chains/v11/chains_dev.json

```
@@ -7711,5 +7711,89 @@
         "options": [
             "testnet"
         ]
+    },
+    {
+        "chainId": "eip155:2088",
```

I suppose there should be:
```suggestion
        "chainId": "eip155:aa3876c1dc8a1afcc2e9a685a49ff7704cfd36ad8c90bf2702b9d1b00cc40011",
```

---

done in both